### PR TITLE
PP-6029 Node buildpack manifest

### PR DESCRIPTION
**Pull request by mrlumbu**
Adds node buildpack manifest

To test run e.g.
```
cf push --vars-file dev.yml \
--var adminusers_url=http://adminusers-dev-dom.apps.internal \
--var  products_url=products-dev-dom.london.cloudapps.digital \
--var  publicauth_url=publicauth-dev-dom.london.cloudapps.digital \
--var direct_debit_connector_url=http://directdebit-connector-dev-dom.apps.internal \
--var direct_debit_frontend_url=pay-directdebit-frontend-dev-dom.london.cloudapps.digital
```

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    NODE_ENV: production
+    SESSION_ENCRYPTION_KEY: ((direct_debit_frontend_session_encryption_key))
+    COOKIE_MAX_AGE: '10800000'
+    ANALYTICS_TRACKING_ID: ((direct_debit_frontend_analytics_tracking_id))
+    ANALYTICS_TRACKING_ID_XGOV: ((direct_debit_frontend_analytics_tracking_id_xgov))
+    CONNECTOR_URL: ((direct_debit_connector_url))
+    PRODUCTS_URL: ((products_url))
+    PUBLICAUTH_URL: ((publicauth_url))
+    SENTRY_DSN: ((direct_debit_frontend_sentry_dsn))
+    ENVIRONMENT: ((environment))
```
Comment (vixus0) [inline manifest.yaml]:
```suggestion
    ENVIRONMENT: ((space))
```
Minor nitpick but `((space))` would be more consistent with the other apps.

Comment (mrlumbu) [reply manifest.yaml]: ah, yes, good point

File: dev.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+memory: 500M
+disk_quota: 500M
+direct_debit_frontend_session_encryption_key: asdjhbwefbo23r23rbfik2roiwhefwbqw
+direct_debit_frontend_analytics_tracking_id: testing-123
+direct_debit_frontend_analytics_tracking_id_xgov: testing-123
+environment: dev
```
Comment (richardTowers) [inline dev.yml]: Should this be `space` now as well?

Comment (mrlumbu) [reply dev.yml]: yes 😬

**Pull request merged**